DCode.Web/Scripts/App_JS/Controllers/Contributor.js

```diff
@@ -45,6 +45,11 @@
             { Id: 2, Description: "Firm Initiative" },
             { Id: 3, Description: "Industry Initiative" }];
 
+        $scope.selectProfTypes = [

```

Comment:
Can we please pull the values directly from the database. Master values should reside at only one location.
Also, please rename as `$scope.selectProficiencyTypes`.